doc/page-cmakefunctions.h

```diff
@@ -114,6 +136,8 @@ This function adds a test with the name \e TargetName.The parameter descriptions
 \param SOURCES_GLOB_RECURSE List of source files searched recursively inside folder and subfolders, used to build this target provided as a wildcard expression
 \param HEADERS_GLOB_RECURSE List of header files to be MOCed and added to this target, searched recursively inside folder and subfolders provided as a wildcard expression
 \param LINK_LIBRARIES List of libraries that will be linked with this target
+\param HEADERS_NO_MOC The headers that should be excluded from moc processing
+\param HEADERS_NO_MOC_FILE The list of the files that should be excluded from moc processing
```

Comment:
If you introduce that , you also need to introduce HEADERS_FILE, HEADERS_NO_INSTALL_FILE.
But what problem is it supposed to solve ? Did you experience any limitation with CMake regarding lists length ??

Reply:
>HEADERS_NO_INSTALL_FILE, SOURCES_FILE, etc....

sure, if we need it, then why not, but HEADERS_NO_MOC_FILE is enough for now.

> But what problem is it supposed to solve ? Did you experience any limitation with CMake regarding lists length ??

in this PR i'm processing and iterating HEADERS_NO_MOC_GENERATED
and removing such files from HEADERS_FOR_MOC.
But what if the number of non-MOC files is too large and we use **facelift_add_library** outside of Facelift, where can we define this list?
File is the best choice, and no extra options like HEADERS_NO_INSTALL_FILE are needed !